blackjack/blackjack_test.go

```diff
@@ -112,83 +136,84 @@ func TestPlayerBust(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	g.Player.Action = blackjack.ActionHit
+	players := []blackjack.Player{
+		{Name: "Planty"},
+		{Name: "Kevin"},
+		{Name: "Donald"},
+	}
+
+	g.Players = players
+
+	g.Players[0].Action = blackjack.ActionStand
+	g.Players[1].Action = blackjack.ActionStand
+	g.Players[2].Action = blackjack.ActionHit
+
 	g.Start()
 
-	got := blackjack.ReportMap[g.Player.HandOutcome]
+	output := &bytes.Buffer{}
+
+	g.Outcome(output)
 
-	want := "***** Bust!  Player loses *****"
+	want := "BlackjackWinBust"                                                                                       //"Planty: ***** Blackjack!  Player wins *****\nKevin: ***** Player wins! *****\nDonald: ***** Bust!  Player loses *****\n"
+	got := g.Players[0].HandOutcome.String() + g.Players[1].HandOutcome.String() + g.Players[2].HandOutcome.String() //output.String()
 
 	if want != got {
-		t.Fatalf("wanted: %s, got: %s", want, got)
+		t.Fatalf("wanted:%q, got %q", want, got)
 	}
 
 }
 
-func TestOnlyStandAI(t *testing.T) {
+func TestRemoveQuitPlayers(t *testing.T) {
 	t.Parallel()
 
-	stack := []cards.Card{
-		{Rank: cards.Ace, Suit: cards.Club},
-		{Rank: cards.Seven, Suit: cards.Club},
-		{Rank: cards.Jack, Suit: cards.Club},
-		{Rank: cards.Queen, Suit: cards.Club},
+	g := blackjack.Game{
+		Players: []blackjack.Player{
+			{Action: blackjack.ActionStand},
+			{Action: blackjack.ActionQuit},
+			{Action: blackjack.ActionQuit},
+		},
+	}
 
-		{Rank: cards.Ten, Suit: cards.Heart},
-		{Rank: cards.King, Suit: cards.Heart},
-		{Rank: cards.Two, Suit: cards.Heart},
-		{Rank: cards.Queen, Suit: cards.Heart},
+	g.Players = g.RemoveQuitPlayers()
```

Comment:
I'm not sure if it's clear why we need this. Can you elaborate a bit on what `RemoveQuitPlayers` is for?

Reply:
This is when a multiplayer game is in session and a player quits the game.  I opted to remove the quitting player from the Players []Player and then have the game continue for the remaining players.  I'm not sure if this is the best approach though.  What do you think?

Reply:
We could try something like:

```go
p.Quit()
```

on the Player object.